tictactoe.js

```diff
@@ -1,105 +1,151 @@
-function Game(){
-	this.playerO = {
-		symbol: "O",
-		number: 0
-	};
-	this.playerX = {
-		symbol: "X",
-		number: 1
-	};
-	this.players = [this.playerO, this.playerX],
+function Player(symbol) {
+	this.symbol = symbol;
+}
 
-	this.board = {
-		grid: [],
-		init: function() {
-			this.grid = [["","",""], ["","",""], ["","",""]];
-		},
-		display: function() {
-			for (var i = 0; i < 3; i++) {
-				for (var j = 0; j < 3; j++) {
-					var cell = document.getElementById(String(i) + String(j));
-					cell.textContent = this.grid[i][j];
-				}
-			}
-		}
-	};
+var playerX = new Player("X");
+var playerO = new Player("O");
 
-	this.checkWin = function(player) {
-		var symbol = player.symbol;
-		var grid = this.board.grid;
+playerX.nextPlayer = playerO;
+playerO.nextPlayer = playerX;
+
+var board = {
+	grid: [],
+	init: function() {
+		this.grid = [["","",""], ["","",""], ["","",""]];
+	},
+	display: function() {
 		for (var i = 0; i < 3; i++) {
-			//check horizontals
-			if (grid[i][0] === symbol &&
-					grid[i][1] === symbol &&
-					grid[i][2] === symbol) {
-				return true;
-			}
-			//check verticals
-			if (grid[0][i] === symbol &&
-					grid[1][i] === symbol &&
-					grid[2][i] === symbol) {
-				return true;
+			for (var j = 0; j < 3; j++) {
+				var cell = document.getElementById(String(i) + String(j));
```

Comment:
I really like this way of finding the grid elements. So each has an id of xy? Great Job.